admin/sql/create_tables.sql

```diff
@@ -8,4 +8,14 @@ CREATE TABLE "user" (
 );
 ALTER TABLE "user" ADD CONSTRAINT user_musicbrainz_id_key UNIQUE (musicbrainz_id);
 
+CREATE TABLE listens (
+  id              SERIAL,
+  uid             VARCHAR NOT NULL,
+  ts              TIMESTAMP WITH TIME ZONE NOT NULL,
+  artist_msid     UUID NOT NULL,
+  album_msid      UUID,
+  recording_msid  UUID NOT NULL,
+  json            VARCHAR
```

Comment:
The raw data submitted by the user is stored there -- the user is allowed to put other pieces of data into this space that we can't know to stick into the DB. This is all good.